NDTensors/src/lib/UnallocatedArrays/src/abstractfill/abstractfill.jl

```diff
@@ -9,7 +10,10 @@ end
 
 ## TODO this fails if the parameter is a type
 function alloctype(Atype::Type{<:AbstractFill})
-  return get_parameter(Atype, Position{4}())
+  return parameter(Atype, 4)
 end
 
-set_axestype(T::Type{<:AbstractFill}, ax::Type) = set_parameters(T, Position{3}(), ax)
+axestype(t::Type{<:AbstractFill}) = parameter(t, 3)
```

Comment:
```suggestion
axestype(t::Type{<:AbstractFill}) = parameter(t, axestype)
```

Reply:
I added the infrastructure to make this work and updated the code, thanks!